GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -402,8 +359,28 @@ private static string GetRegistryValue(RegistryKey root, string subkey, string k
             return value ?? string.Empty;
         }
 
-        private static string GetVisualStudioPath()
+        private static string GetVsDiffMergePath()
         {
+            //%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe
+            var vsVersionNumber = new string[] { "2020", "2019", "2018", "2017" };
```

Comment:
Nowhere ;) I extrapolate the pattern to not be obliged to wait months before someone notice, open an issue and a new release is done to have the feature working.

For example, noone opened an issue telling that that do not works with VS2017. And I found it because I made a refactoring in this file add another editor....